common/components/Header/components/TabsOptions.jsx

```diff
@@ -62,7 +63,7 @@ export default class TabsOptions extends Component {
                             {
                                 tabs.map((object, i) => {
                                         // if the window pathname is the same or similar to the tab objects name, set the active toggle
-                                        const activeOrNot = (window.location.pathname === object.name || window.location.pathname.substring(1) === object.name) ? 'active' : '';
+                                        const activeOrNot = (window.location.pathname === object.link || window.location.pathname.substring(1) === object.link) ? 'active' : '';
```

Comment:
better use router props.location object, that way we could make it work in non-browser contexts